package.json

```diff
@@ -25,6 +25,7 @@
     "build": "lerna run build --scope @globalfishingwatch/*",
     "test": "lerna run test --scope @globalfishingwatch/*",
     "test:prepare": "lerna run prepare --scope @globalfishingwatch/*",
+    "test:vessel-history": "cross-env lerna run test --scope @globalfishingwatchapp/vessel-history",
```

Comment:
I don't think so... but would there be a way to pass the test:`:[command]` as a variable ? so we avoid duplicating the same command for every project, anyway it looks good

Reply:
Not that I know of...